packages/components/src/DateTimePickers/pickers/TimePicker/index.js

```diff
@@ -1,3 +1,6 @@
-import TimePicker from './TimePicker.component';
+import TimePicker, { twoDigits } from './TimePicker.component';
 
 export default TimePicker;
+export {
+    twoDigits,
```

Comment:
This is used for Time and date formatting in InputDateTimePicker. You should extract that somewhere, expose it, and import that when needed.
It feels weird to import something from `blabla/TimePicker` to format a date

Reply:
yeah I have think the same of it, it feel wrong, but @jmfrancois is utils-epileptic.
I will do it, it's our little secret ^^

Reply:
Just don't name the things `utils` but with meaningful names related with the methods you expose.
Something like `formatter.js`

Reply:
✔